### ISSUE TITLE
[COGNITION]: VAOS - CONSIDER making the Past Appointments update button enabled at all times

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#8940]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️  Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The Past Appointments has an update button that is disabled until users make a choice from the date range dropdown. I think this could be confusing to screen reader users because we're not offering instructions how to enable the button. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Date range button is enabled at all times
- [ ] No axe errors appear after the update is made

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-05-08 at 10 17 24 AM](https://user-images.githubusercontent.com/934879/81449933-f86f3f80-9146-11ea-91d6-876fbfd9c513.png)

**Comment by rtwell** [comment]
great question @1Copenut …it probably should always be enabled. I'm also curious about the placement of this—is top of page better or bottom? I assumed the bottom would be better because once you get to the bottom of the list and you cannot find what you're looking for, that's when you want to show more. then again, at least the expectation is set at the top of the page "showing last 3 months" and you know if you want to see something from 6 months ago, you dont have to wonder if it's in this list. hmmmm…

**Comment by 1Copenut** [comment]
@rtwell I've seen this pattern almost exclusively at the top of the page. That's not to say it's the only way, but it's one I think has a level of familiarity. Maybe an analytics event on the `Update` button and A/B test top v. bottom placement? Def. just tossing ideas here.

**Comment by rtwell** [comment]
let's go with top o' the page! tyvm

**Comment by jbalboni** [comment]
@rtwell @1Copenut The reason I disabled it at times was to provide some contextual info about what range is currently being displayed, now that the select box doesn't provide that. There's no way currently for a user to clearly see what the range being shown is, that might be worth addressing as part of this change.

**Comment by rtwell** [comment]
ah, yes, i was thinking about this, but you articulated it very clearly—let's treat this like our search result pattern: whatever is default, we should print a little header that reads "Displaying the past 3 months of appointments" and then if you switch to 6 months, that little header changes to the appropriate range. thoughts?

**Comment by jbalboni** [comment]
@rtwell That sounds good to me. Can you mock it up? I'm not sure where it would go in relation to the existing "Past appointments" header.

**Comment by rtwell** [comment]
@jbalboni I was thinking something like this, but now that i see it, is it not redundant since the select action is also displaying the date range?

![image](https://user-images.githubusercontent.com/12564977/81839813-10174100-9516-11ea-9c65-d9049b9e1e0e.png)

**Comment by jbalboni** [comment]
@rtwell The select won't always match the appointments being show. When a user changes the range, it'll be out of sync with what's show until they click Update.

**Comment by rtwell** [comment]
true…but what they are seeing (the list of appointments) will also not change until they click update, right? although, it would depend on the range—if we first show present day–3 months, and then 3–6 months, and then 6–9 months (3 month chunks at a time) vs a cumulative 3 months, then show 6 months (the 3 currently displayed + next three), then 9 months, etc, the appointment(s) within view may or may not change.

**Comment by jbalboni** [comment]
@rtwell Sorry, lost track of this. Yeah, I think you may or may not see the appointment list change (especially the first couple) when you're choosing different range options. It just seems confusing to not have a header, label, or some indication of what precisely is being shown.

**Comment by rtwell** [comment]
cool, as long as the header is helpful, let's use it

**Comment by jenniemc** [comment]
Hey @lenaecb, I'm picking up this tix from the Ready column and unassigned you.
